src/framework/shortcuts/view/shortcutsmodel.cpp

```diff
@@ -163,6 +188,15 @@ QVariant ShortcutsModel::currentShortcut() const
     return shortcutToObject(sc);
 }
 
+QString ShortcutsModel::getCategory(ui::ActionCategory category) const
+{
+    if ((int)category + 1 >= 0 && (int)category + 1 < categories.size()) {
+        return categories[(int)category + 1].qTranslated();
+    }
+//    LOGE() << "Categories: " << categories.size() << "| Requested Category: " << (int)category << " by: " << code;
+    return "NULL";
+}
```

Comment:
Thanks for moving the function, but why did you go back to using C-style casts? It should be:
```C++
    QString getCategoryName(ui::ActionCategory category) const
    {
        size_t c = static_cast<int>(category) + 1;
        IF_ASSERT_FAILED(c < categories.size()) {
            return QString();
        }
        return categories[c].qTranslated();
    }
```

Reply:
Ah, so sorry, must have missed it during handling a merge conflict